.github/workflows/triage.yml

```diff
@@ -0,0 +1,61 @@
+name: Add new issues to triage column
+
+on:
+  issues:
+    types:
+      - opened
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This seems unused.

Reply:
No, it is required by the GitHub cli: https://github.com/cli/cli/blob/e7102f9d84dae8135aca13edffe4c7d0049bdde4/cmd/gh/main.go#L317
I set it to the entire workflow because both steps needed it. But it can be scoped for each step.